IFC 4.3/Alignment_3/UT_Alignment_3.ifc

```diff
@@ -0,0 +1,57 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'export\\Simple_Alignment.ifc',
+/* time_stamp */ '2020-11-20T09:35:52',
+/* author */ ('stefan'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.1.0 by Geometry Gym Pty Ltd built 2020-11-08T11:27:43',
+/* originating_system */ 'GeometryGymIFC v0.1.1.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3RC1'));
```

Comment:
Please update to `IFC4x3_RC2` together with the corresponding alignment entities.

Reply:
Resolved in 9b0375a

Reply:
Resolved in
b4195aa